x/ecocredit/server/basket/helpers.go

```diff
@@ -0,0 +1,48 @@
+package basket
```

Comment:
if we're copying this from `data_prefixes.go` we should just move it to `keys.go` and share it between the two packages